server/middlewares/auth.js

```diff
@@ -0,0 +1,13 @@
+const { verify } = require('jsonwebtoken');
+
+require('env2')('config.env');
+
+const isAuthorized = (req, res) => {
+  verify(req.cookies.applicant, process.env.SECRET_KEY, (err, decoded) => {
+    if (err) {
+      res.status(401).json({ msg: 'you are not authorized' });
+    }
+  });
+};
```

Comment:
you should create a `token` in else check then call next() method in the same block
```suggestion
const isAuthorized = (req, res , next ) => {
  jwt.verify(req.cookies.applicant, process.env.SECRET_KEY, (err, token) => {
    if (err) {
      res.status(401).send({ statusCode: 401, error: 'you are un authorized' });
    }
  });
};
```